fix: How to get Logs for Support (#760)
appsmithorg/appsmith-docs: website/docs/getting-started/setup/installation-guides/docker/README.mdx

1. Create an installation folder called `appsmith` for deployment and data storage.
2. `cd` into the installation folder.

> ## Setup with Docker compose _(recommended)_

Reviewer: Please keep it as before. Docker Compose is the name of the tool

Author: done

## Docker logs

> Once the Appsmith container is ready, the message "Appsmith is Running" would be displayed in the Docker logs.

Reviewer: ```suggestion
Once the Appsmith container is ready, the message "Appsmith is Running!" is displayed in the Docker logs.
```

docker logs -f appsmith
```

> This message is also logged and available in server logs in (`stacks/logs/backend/backend.log`). You can access backend logs from your Appsmith deployment by following these steps:

Reviewer: ```suggestion
Backend/Server logs are stored in `stacks/logs/backend/backend.log`. You can access backend logs from your Appsmith deployment by following these steps:
```

> 

Reviewer: I think the below steps can be shown in a video. cc: @jnikhila what do you think?

Reviewer: in fact we can just link to felix's document - https://github.com/felix-appsmith/guide-how-to-get-logs-appsmith

Author: I think the ticket mentioned that we need an official doc in docs.appsmith

Reviewer: i agree but in our docs we don't need to put a context around how the log files can be shared with the support team. So for eg the line "Retrieve the .log files and bundle them in a .zip file." sounds a bit  odd as the reader does not understand the context. We should definitely mention the backend logs which was already there, but for some reason it was not discoverable because of that single line statement right under front end logs.

* Navigate to the folder where Appsmith was deployed, open the `logs` folder within the `stacks` folder.
* Find the `backend` folder.
* Retrieve the `.log` files and bundle them in a `.zip` file. 

> You can either share these logs with the Appsmith team or retain them for your own reference.

Reviewer: remove this line

Author: done
